app/src/main/java/com/example/stretchy/features/datatransport/DataExporter.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.stretchy.features.datatransport
+
+interface DataExporter {
+    suspend fun saveDataInFile()
+}
```

Comment:
`saveDataInFile()` -> `exportData`

The file thing is an implementation details, you design interface, the implementation does not need to be file, it could be a cloud solution in future for example